setup.cfg

```diff
@@ -14,9 +14,9 @@ project_urls =
 
 classifiers =
     Development Status :: 4 - Beta
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
```

Comment:
I guess `python_requires = >=3.8` can also be changed to 3.10? Or was it intentional to keep it at 3.8?

Reply:
Yes you are absolutely right not sure how I missed that.